app/interactors/withdraw/create_interactor.rb

```diff
@@ -39,10 +42,26 @@ def check_for_issues
     context.fail!(issues: issues) if issues.any?
   end
 
-  def withdraw
+  def check_previous_withdrawal
+    context.already_withdrawn = edition.withdrawn?
+    return unless already_withdrawn
+
+    previous_explanation = edition.status.details.public_explanation
+    context.fail!(unchanged: true) if previous_explanation == params[:public_explanation]
```

Comment:
I think this unchanged may be superfluous - unless I missed something.

Reply:
True - I had a vague intention of referring to it in the controller. Removed.